pages/dkp/kommander/2.2/DKP-insight/index.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: DKP Insight
+title: DKP Insight
+excerpt: Use DKP insight to monitor and tune your enviornment
```

Comment:
```suggestion
excerpt: Use DKP insight to monitor and tune your environment
```
Just a typo here.

Reply:
Please use DKP Insights instead of DKP insight